### DISCUSSION TITLE
How to show/hide ironbar?

### DISCUSSION THREAD

**Discussion by preetamsad** [JakeStanger/ironbar#285] [Q&A] [answered]
Is it not possible to toggle bar's visibility using the `set-visible` option? The name suggests i should be able to set the bar's visibility to true/false but it takes only one argument the bar-name .

So, I am doing `ironbar set-visible coolbar`  and it hides the bar if it is visible, but running the command again does not shows the bar,  it just prints `ok`.

 Is it supposed to work that way only? or am I doing something wrong?

**Comment by JakeStanger** [comment, marked as answer]
I'm afraid I'm not at my PC and won't be for a while so this is from memory, but I believe there is a --visible (or otherwise named) switch you can pass to toggle it to true.

I will likely look at bringing this in line with other commands to show/hide/toggle as part of the next round of updates.

**Reply by preetamsad** [reply]
Yes `ironbar set-visible coolbar --visible` works fine. Thanks.

**Reply by CG-GeisT** [reply]
How would you go about doing this with the current version of ironbar? The ironbar set-visible -- visible doesn't work for me.

**Reply by JakeStanger** [reply]
This has changed to `ironbar bar set-visible`. You can use `--help`, including on any (sub)command to see details.
